### DISCUSSION TITLE
Suggestion: move the Next/Prev buttons and the "total rows" text outside of the table so they are always visible

### DISCUSSION THREAD

**Discussion by tmaszk** [tfwright/live_admin#107] [Ideas]
When viewing a list of records it would be nice to have access to the Next and Prev buttons and see the "xx total rows" without having to horizontally scroll.  Notice that the area above the table is always visible (i.e. where the table name and search box are).

**Screenshot examples.**
The Next and Prev buttons are visible when the table is scrolled all the way to the left. You can't see the number of rows.
![Screenshot 2024-03-13 at 9 32 09 AM](https://github.com/tfwright/live_admin/assets/4540522/9b49a3ac-32a9-41e1-8e44-fec2cf49e81d)

While scrolling horizontally, you can't see either the buttons or the total rows
![Screenshot 2024-03-13 at 9 32 37 AM](https://github.com/tfwright/live_admin/assets/4540522/acc28e21-515a-48c3-8cab-54b4131ae862)

The "total rows" text is only visible when the table is scrolled all the way tot he right.  You can't see the Next/Prev buttons.
![Screenshot 2024-03-13 at 9 32 54 AM](https://github.com/tfwright/live_admin/assets/4540522/4d310da3-1a64-45ea-a889-673e1c2cc592)

**Comment by tfwright** [comment]
Suggestion implemented in https://github.com/tfwright/live_admin/commit/4ac35598f59c3a493345640608022d9554ae906e
